.changeset/tricky-llamas-call.md

```diff
@@ -1,5 +1,5 @@
 ---
-'@shopify/cli-hydrogen': patch
+'@shopify/cli-hydrogen': minor
```

Comment:
We only do patches

Reply:
Wait that's only in hydrogen and hydrogen-react. The CLI can have minors, right?

Reply:
oh whoops right